QA add test PCP-4382 and PCP-4400

sedatbsr opened the pull request:
Tests added:

**- PCP-4382**
**- PCP-4400**

Commit:
Tests added:
PCP-4382
PCP-4400

mishautkin commented on tests/qa/tests/02-dashboard-ui/visual-onboarding-ui.spec.ts:
```diff
@@ -147,3 +147,54 @@ test.describe( '', () => {
+		} );
+
+		await pcpOnboarding.visit();
+		await pcpOnboarding.page.reload();
```
Why `page.reload()`?

mishautkin commented on tests/qa/tests/02-dashboard-ui/visual-onboarding-ui.spec.ts:
```diff
@@ -147,3 +147,54 @@ test.describe( '', () => {
+		await pcpOnboarding.page.reload();
+		await pcpOnboarding.gotoInitialOnboardingPage();
+		await pcpOnboarding.snapshotLocator( pcpOnboarding.onboardingContentContainer(),
+			`${ testInfo.title }WooPayments - Onboarding Initial Page - Serbia`, {timeout: 3000}
```
I think it's ok to go with simply `${ testInfo.title }` in this case, since the test key and all info is already present in test summary.

mishautkin commented on tests/qa/tests/02-dashboard-ui/visual-onboarding-ui.spec.ts:
```diff
@@ -147,3 +147,54 @@ test.describe( '', () => {
+	}, testInfo ) => {
+		await pcpOnboarding.visit();
+		await pcpOnboarding.snapshotLocator( pcpOnboarding.onboardingContentContainer(),
+			`${ testInfo.title } - WooPayments - Onboarding Initial Page`, {timeout: 3000}
```
Should we, maybe, add WooPayments to the test summary? Like in the second TC?
`${ testInfo.title } - Onboarding Initial Page`
Screenshot names would be:
"PCP-4382 | WooPayments - Settings - Onboarding - Default UI - Onboarding Initial Page"
"PCP-4382 | WooPayments - Settings - Onboarding - Default UI - Product Types"
What do you think?

mishautkin reviewed (commented):
Added few comment on SSs naming

Commit:
fixes and updates

screenshot names updated
page.reload() removed
test titles updated (also on doc)
